docs/general/faq/community-faqs.md

```diff
@@ -0,0 +1,349 @@
+
+### **What's the difference between a full node and an archive node?**
+
+A full node only saves all block headers and transactions from the most recent era. An archive node will save all block headers and transactions.
+
+### **How to check the reason for transaction failure?**
+
+The **`txExecErrorMsg`** in the transaction receipt provides the error information for the transaction.
+
+### **What are the situations for tx revert?**
+
+Tx revert means the contract execution failed. Contract developers should use **`require`** with an error message where an exception might occur. This way, the contract execution error will be seen in the **`txExecErrorMsg`** of the transaction receipt.
+
+### **What does this error mean? `Failed imported to deferred pool: Tx with same nonce already inserted. To replace it, you need to specify a gas price > 20000000000`**
+
+There is already a transaction with the same nonce. Either wait for it to be packaged or send a new one with the same nonce, but you need to raise the gasPrice.
+
+### **Is there a method in js-conflux-sdk to parse the data in tx?**
+
+First, initialize the contract using abi, then use **`contract.abi.decodeData()`**.
+
+### **Can the logger of the Conflux class in js-conflux-sdk use something other than console?**
+
+Any object that has implemented the methods error, info, and log can be used.
+
+### **If you use a sponsorship contract, does it mean that all users operate the contract, regardless of which method of the contract is called, are they all paid according to a unified standard?**
+
+Yes.
+
+### **Once the contract is set up for sponsorship, does it mean that all methods use the balance of the sponsorship? Is there a way to specify a method to skip sponsorship?**
+
+The current version does not support this.
+
+### **Is there any information on the conflux sponsorship mode?**
+
+[Here's the link](../../core/learn/core-space-basics/sponsor-mechanism).
+
+### **Conflux-studio can't run on Windows 10 system.**
+
+[Here's the forum link for reference](https://forum.conflux.fun/t/topic/4280).
+
+### **Are ERC20/ERC777 still called this way in the Conflux network?**
+
+They are still referred to as ERC20/ERC777, without any separate standard.
+
+### **What are the websocket service ports of the official nodes of the mainnet and testnet?**
+
+Mainnet URL: ws://main.confluxrpc.org/ws
+Testnet URL: ws://test.confluxrpc.org/ws
+
+### **When `docker pull confluxchain/conflux-rust` prompts "no such file", how to solve it?**
+
+It might be a network issue. You can try setting a registry for docker:
+
+```json
+
+{
+  "registry-mirrors": [
+    "https://at8ak49f.mirror.aliyuncs.com"
+  ],
+  "experimental": false,
+  "debug": true}
+
+```
+
+### **When we use the wallet to transfer, we are asked to enter the set password, and then the transfer is successful. Why use symmetric encryption instead of a private key here?**
```

Comment:
I think this is not a good writing question and so as the answer. Suggest moving it